README.md

```diff
@@ -240,6 +240,19 @@ parameters:
                 # if credentials parameter is not empty, it will be used instead of this file.
                 # this file must not be committed into the repository!
                 credentialsFilePath: null
+
+            youtrack:
+                # e.g. https://your-company.youtrack.cloud
+                server: https://acme.youtrack.cloud
+
+                # see below for possible formats.
```

Comment:
Does "see below for possible formats" make sense here? Unlike Jira, here the only possible format is just a simple string containing the permanent token, am I right? Then it could be a little confusing for readers, so maybe this should be just `# YouTrack permanent token`?

Reply:
Replaced with `# YouTrack permanent token` indeed